### ISSUE TITLE
Can someone continue maintain this project ?

### ISSUE THREAD

**Issue by portsip** [ipkn/crow#344]
Really like it, @ipkn it's possible let other contributor maintain it if you didn't have time ?

**Comment by grober82** [comment]
He is too busy to answer that simple question.

**Comment by conradjones** [comment]
@portsip  you can just fork it on github and maintain it there

**Comment by kingsamchen** [comment]
@mrozigor May I ask that are you still maintaining your fork of crow?

**Comment by mrozigor** [comment]
Sorry, but don't have time to create new things. Tried to address some issues, but that's all. If you want to commit something to my fork, then feel free - I'll try to look at this and merge it.

**Comment by kingsamchen** [comment]
一次次 有&

获取 Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kingsamchen** [comment]
获取 Outlook for Android<https://aka.ms/ghei36>
还点按剪贴内容即可将其粘贴到文本框中。
________________________________
点按剪贴内容即可将其粘贴到文本框中。
发件人: Igor Mroz <notifications@github.com>
发送时间: 2020年5月17日星期日 21:42
收件人: ipkn/crow
抄送: Kingsley Chen; Commen4
主题: Re: [ipkn/crow] Can someone continue maintain this project ? (#344)


Sorry, but don't have time to create new things. Tried to address some issues, but that's all. If you want to commit something to my fork, then feel free - I'll try to look at this and merge it.

―

**Comment by DavidUser** [comment]
The really sad notice about this project slowdown is that it has more stars and forks than most of other REST API frameworks in C++
It’s simple a huge potential lost 😔

**Comment by The-EDev** [comment]
@DavidUser well it's not really lost if people are willing to support it, @ipkn is one person, just because he stopped working on the project doesn't mean it's done for, I'd suggest finding whatever bugs or issues and adding them yourself, then making a PR to @mrozigor's repo since he has the most recent commit.

**Comment by ayaankhan98** [comment]
@The-EDev yes you are right, nothing is lost we can make proper community for the maintainance of this project and for generating proper documentaion.

@DavidUser @mrozigor @portsip @ipkn
